### ISSUE TITLE
CounterReponseStream throws IllegalStateException when using Jetty

### ISSUE THREAD

**Issue by squierts** [javamelody/javamelody#488]
If a filter in the chain has already called getWriter() then the Counter Servlet will throw an exception because it's trying to call getOutputStream() which apparently isn't allowed if getWriter() was already called. 
```
java.lang.IllegalStateException: WRITER
        at org.eclipse.jetty.server.Response.getOutputStream(Response.java:681)
        at net.bull.javamelody.CounterResponseStream.<init>(CounterResponseStream.java:38)
        at net.bull.javamelody.CounterServletResponseWrapper.createOutputStream(CounterServletResponseWrapper.java:75)
        at net.bull.javamelody.FilterServletResponseWrapper.getOutputStream(FilterServletResponseWrapper.java:109)
```

**Comment by evernat** [comment]
Good catch, thanks.
This is fixed in master (1c6df051892769398917d78a96d7e35beba674e3) and ready for the next release (1.57).

**Comment by squierts** [comment]
FYI I confirmed the fix in my setup
